example/package.json

```diff
@@ -13,16 +13,16 @@
   "scripts": {
     "clean": "rimraf dist",
     "start": "electron .",
-    "exe32": "electron-packager . poopie --platform linux --arch ia32 --out dist/ --ignore \"(dist|node_modules/electron.*)\"",
-    "exe64": "electron-packager . poopie --platform linux --arch x64 --out dist/ --ignore \"(dist|node_modules/electron.*)\"",
+    "exe32": "electron-packager . poopie --platform linux --arch ia32 --out dist/",
+    "exe64": "electron-packager . poopie --platform linux --arch x64 --out dist/",
     "deb32": "electron-installer-debian --src dist/poopie-linux-ia32/ --arch i386 --config config.json",
     "deb64": "electron-installer-debian --src dist/poopie-linux-x64/ --arch amd64 --config config.json",
     "build": "npm run clean && npm run exe32 && npm run deb32 && npm run exe64 && npm run deb64"
   },
   "devDependencies": {
-    "electron-installer-debian": "*",
-    "electron-packager": "*",
-    "electron-prebuilt": "*",
-    "rimraf": "*"
+    "electron-installer-debian": "^0.6.0",
+    "electron-packager": "^9.0.0",
+    "electron": "^2.0.0",
```

Comment:
This has to be v1.7.8, otherwise, the example is not working when people try building it. v2.0.0 has not been released yet so `npm install` shows errors. We could also use beta version v1.8.1 but I think the example should be a stable and reliable version. This will also help for people running the example for debugging errors